pom.xml

```diff
@@ -266,7 +267,7 @@
         <gravitee-resource-cache-redis.version>3.0.0-alpha.1</gravitee-resource-cache-redis.version>
         <gravitee-resource-oauth2-provider-keycloak.version>2.1.0</gravitee-resource-oauth2-provider-keycloak.version>
         <gravitee-service-geoip.version>3.0.0</gravitee-service-geoip.version>
-        <gravitee-secretprovider-kubernetes.version>1.0.1</gravitee-secretprovider-kubernetes.version>
+        <gravitee-secretprovider-kubernetes.version>2.0.0-APIM-7497-api-secrets-SNAPSHOT</gravitee-secretprovider-kubernetes.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:** etprovider-kubernetes.version>************************************</gravitee-secretpro  
[This comment was created by Aqua Pipeline]